include/psp2/kernel/modulemgr.h

```diff
@@ -28,27 +28,28 @@ extern "C" {
 /** @} */
 
 typedef struct SceKernelSegmentInfo {
-	SceUInt size;   //!< sizeof(SceKernelSegmentInfo)
-	SceUInt perms;  //!< probably rwx in low bits
-	void *vaddr;    //!< address in memory
-	SceUInt memsz;  //!< size in memory
-	SceUInt flags;  //!< meaning unknown
-	SceUInt res;    //!< unused?
+  SceSize size;   //!< this structure size (0x18)
+  SceUInt perms;  //!< probably rwx in low bits
+  void *vaddr;    //!< address in memory
+  SceSize memsz;  //!< size in memory
+  SceSize filesz; //!< original size of memsz
+  SceUInt res;    //!< unused
 } SceKernelSegmentInfo;
 
 typedef struct SceKernelModuleInfo {
-	SceUInt size;                       //!< 0x1B8 for Vita 1.x
-	SceUInt handle;                     //!< kernel module handle?
-	SceUInt flags;                      //!< some bits. could be priority or whatnot
+	SceSize size;                       //!< 0x1B8 for Vita 1.x
+	SceUID modid;
+	uint16_t modattr;
+	uint8_t  modver[2];
 	char module_name[28];
 	SceUInt unk28;
-	void *module_start;
-	void *module_stop;
-	void *module_exit;
+	void *start_entry;
+	void *stop_entry;
+	void *exit_entry;
 	void *exidxTop;
 	void *exidxBtm;
-	SceUInt unk40;
-	SceUInt unk44;
+	void *extabTop;
+	void *extabBtm;
```

Comment:
sorry, last thing.
we already break the backward compatibility.
could you change these `camelCase` to `snake_case` ?